package.json

```diff
@@ -68,6 +68,7 @@
     "onetime": "^5.1.0",
     "p-limit": "^2.2.0",
     "pumpify": "^2.0.0",
+    "readable-stream": "^3.4.0",
```

Comment:
I have been playing whack-a-mole trying to avoid bringing that package in, and instead rely on it being in node.js core.  What versions of node.js don't currently work with async iterators?

Reply:
`readable[Symbol.asyncIterator]()` was introduced in Node 10.

Since it all originates from using `through2` ([here](https://github.com/AVaksman/nodejs-storage/blob/support_asyncIterator/src/file.ts#L1194) and [here](https://github.com/AVaksman/nodejs-storage/blob/support_asyncIterator/src/file.ts#L1722)) I am looking in to replacing `through2` with `stream.PassThrough` and dropping `through2` all together.

Reply:
Removed `readable-stream` extra dependency.
Adopt `PassThrough` (with override for `destroy` for Node 8 to mimic Node 10).
Drop `through2` dependency in favor of native `PassThrough`.
WDYT?

Reply:
I defer this one to @bcoe